lib/node_modules/@stdlib/lapack/base/dptts2/lib/index.js

```diff
@@ -0,0 +1,71 @@
+
+/**
```

Comment:
```suggestion
/**
```

Reply:
This is a bug in automation, need to fix it. Opened an issue at https://github.com/Pranavchiku/scripts-stdlib/issues/1